README.md

```diff
@@ -5,7 +5,7 @@
 * Braden S - [ProjectName]()
 * Eli P - [ProjectName]()
 * Isiah W - [ProjectName]() 
-* James C - [ProjectName]() 
+* James C - [CodeCollab](./Mod4ProjectIdeaTemplate.md) 
```

Comment:
The name of the file should be changed, as opposed to using the same name as the template.